### ISSUE TITLE
[WCAG Testing] 1.2.2 Captions (Prerecorded)

### ISSUE THREAD

**Issue by ari-guzzi** [ari-guzzi/ari-guzzi.github.io#8]
**Status:** ❌ Fail
**Severity:** 🟡 Medium

**Notes / Findings**
notes about things

**Recommendation**
woo

**Test Date:** —

_Source: 🌐 WCAG Testing, row 9_
